content/python/concepts/dates/terms/date/date.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: '.date()'
+Description: 'Returns a date in the year-month-day format.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.date()` method returns a date in the year-month-day format.
```

Comment:
Should be clear that it's returning a `date` object type. And list the object properties.